lxd/devices.go

```diff
@@ -72,13 +73,18 @@ func deviceNetlinkListener() (chan []string, chan []string, chan device.USBEvent
 			ueventBuf := make([]byte, r)
 			copy(ueventBuf, b)
 			ueventLen := 0
+			udevEvent := false
 			ueventParts := strings.Split(string(ueventBuf), "\x00")
 			props := map[string]string{}
 			for _, part := range ueventParts {
 				if strings.HasPrefix(part, "SEQNUM=") {
 					continue
 				}
 
+				if strings.HasPrefix(part, "libudev") {
```

Comment:
As pre my previous comment about `udev` vs `kernel` generated uevents. This means that this will only work when `udev` is running on the host. This is usually a reasonable assumption.
If your implementation is depending on `systemd-udevd` to be run on the host you should add a comment about this to the corresponding document. Here it should be sufficient to just mention that you rely on fields from udev itself (Though I'll comment later on whether that's actually needed.).